Automattic/pocket-casts-ios
# Navigates to podcast view when tapping on podcast name on Player.

Fixes #1322

This PR adds the possibility to navigate to the Podcast page when tapping on the podcast name on the Player.


https://github.com/Automattic/pocket-casts-ios/assets/651601/69f3f19d-35bb-40e3-8e43-f774da0461ff



## To test

 - Start the app
 - Start listening to a podcast episode without chapters
 - Open the player full screen
 - Tap on the podcast title
 - Check that the view for podcast opens

## Checklist
